types/three/three-core.d.ts

```diff
@@ -237,7 +237,7 @@ export class AnimationAction {
     reset(): AnimationAction;
     isRunning(): boolean;
     startAt(time: number): AnimationAction;
-    setLoop(mode: AnimationActionLoopStyles, repetitions: number): AnimationAction;
+    setLoop(mode: AnimationActionLoopStyles, repetitions?: number): AnimationAction;
```

Comment:
It looks like `repetitions` isn't optional:

https://github.com/mrdoob/three.js/blob/28894ccbc0787257074dd21cfedacb844bfcca88/src/animation/AnimationAction.js#L134-L141